File: Program243.java
import java.util.*;//for scanner class
import LB.Matrix;

class MyMatrix extends Matrix 
{
    public MyMatrix(int A,int B)
    {
        super(A,B);
    }

    public int CountEven()
    {
        int i=0,j=0,Frequency=0;

        for(i=0;i<Arr.length;i++)
        {
            for(j=0;j<Arr[i].length;j++)
            {
                
                if(Arr[i][j]%2 == 0)
                {
                    Frequency++;
                }
            }
        }
        return Frequency;
    }
}

class Program243
{
    public static void main(String arg[])
    {
        Scanner sobj=new Scanner(System.in);
        int iRow=0,iCol=0,iRet=0;

        System.out.println("Enter number of rows: ");
        iRow=sobj.nextInt();

        System.out.println("Enter number of columns: ");
        iCol=sobj.nextInt();

        MyMatrix mobj=new MyMatrix(iRow,iCol);
        int value=0;

        mobj.Accept();
        mobj.Display();

        iRet=mobj.CountEven();
        System.out.println("even elements are : "+iRet);

        mobj=null;
        System.gc();
    }
}

File: program243.cpp
//convert in generic
#include<iostream>
using namespace std;
template<class T>
class array
{
    public:
    T *Arr;
    int size;

    array(int length);
    ~array();
    void Accept();
    void Display();
    T Addition();
};

template<class T>
array<T>::array(int length)
    {
        size=length;
        Arr=new int[size];
    }

template<class T>
array<T>::~array()
    {
        delete []Arr;
    }

template<class T>
void array<T>:: Accept()
    {
        cout<<"Enter the elements: "<<"\n";
        int icnt=0;

        for(icnt=0;icnt<size;icnt++)
        {
            cin>>Arr[icnt];
        }
    }

template<class T>
void array<T>:: Display()
    {
        cout<<"Elements of the array are: "<<"\n";
        int icnt=0;

        for(icnt=0;icnt<size;icnt++)
        {
            cout<<Arr[icnt]<<"\t";
        }
        cout<<"\n";
    }

template<class T>
T array<T>::Addition()
{
    T sum=0;
    int icnt=0;

    for(icnt=0;icnt<size;icnt++)
    {
        sum=sum+Arr[icnt];
    }
    return sum;
}

int main()
{ 
    array<int> obj(5);
    
    obj.Accept();
    obj.Display();
    int Ret=obj.Addition();
    cout<<"Addition is: "<<Ret<<"\n";

    return 0;
}